catalyst/contrib/runner/alchemy.py

```diff
@@ -8,7 +8,30 @@
 
 class AlchemyRunner(Runner):
     """
-    Runner wrapper with alchemy integration hooks.
+    Runner wrapper with Alchemy integration hooks.
+    Read about Alchemy here https://alchemy.host
+    Powered by Catalyst.Ecosystem
+
+    Example:
+
+        from catalyst.dl import SupervisedAlchemyRunner
+
+        runner = SupervisedAlchemyRunner()
+
+        runner.train(
```

Comment:
the AlchemyRunner doesn't have this method.

only SupervisedAlchemyRunner does

Reply:
correct, so example is based on `SupervisedAlchemyRunner `